src/components/Display.js

```diff
@@ -4,11 +4,6 @@ import PropTypes from 'prop-types';
 function Display(props) {
   const { result } = props;
   return (
-
-  // <div className="Calc">
-  //   <h1>{result}</h1>
-  // </div>
-
     <form action="">
       <input type="text" id="lname" name="lname" className="form-control Calc" value={result} />
```

Comment:
- [ ] All your tests are passing but there is a small error that we should fix before moving on: 
![image](https://user-images.githubusercontent.com/38522578/127914877-c1ef2874-15a7-4563-8c32-fe50321e54b8.png)
I easily fixed this by adding this attribute to the input tag: ```readOnly={true}``` as suggested by the error message, however, I didn't test that the whole app and its features are still working as expected by adding that read-only attribute, please double-check that they do and after that request another code review, well done.